manim/typing.py

```diff
@@ -1,133 +1,462 @@
+"""Custom type definitions used in Manim.
+
+.. admonition:: Note for developers
+    :class: important
+
+    Around the source code there are multiple strings which look like this:
+
+    .. code-block::
+
+        '''
+        [CATEGORY]
+        <category_name>
+        '''
```

Comment:
It's not possible to escape the double quotes like that, it raises an error:
```sh
  File "<unknown>", line 89
    \"\"\"
     ^
SyntaxError: unexpected character after line continuation character
```
I would have also preferred using comments such as `# CATEGORY: <category_name>`. However, the `ast` module ignores `#` comments. It only picks up triple-quote multi-line comments, because `ast` considers them as "expressions" (they're strings after all).

Reply:
Unfortunate, but as I've said, there is no reason to change the already working solution anyways.